documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,116 @@
+# **Metadata Forms**
+
+**Introduction**
+
+Metadata forms allow users to add structured contextual information to various entities in the data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
+
+Metadata forms serve several key purposes:
+
+- Improve data discovery by enabling more consistent, complete, and meaningful metadata
+- Capture domain-specific or organizational metadata standards
+- Streamline metadata management workflows
+- Search among all entities in data.all based on attached metadata
+
+**Metadata Forms visibility**
+
+Visibility setting defines who can view and attach a metadata form to an entity.
+
+- Global visibility means the metadata form is visible and attachable to any entity by all users across the platform
+- Organization/Environment-Wide visibility limits the form to a specific organization/environment - it can only be seen by members of this organization/environment and attached to entities in that organization/environment
+- Team-Only visibility restricts the form to just members of a specific team, but does not restrict to which entities it can be attached to
+
+The same visibility restrictions apply to the attached metadata form. E.g. is userA can see Metadata Form A, but can't see Metadata Forms B, and both of these
```

Comment:
typos: "For example, if userA can see Metadata Form A, but can't see Metadata Form B, and both of these metadata forms are attached to a dataset; then when userA opens the dataset, he will see the attached metadata form A but not the attached metadata form B"